billing/test_main.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import json
+
+import google.auth
+from google.cloud import billing
+import pytest
+
+from main import stop_billing
+
+
+PROJECT_ID = google.auth.default()[1]
+
+
+@pytest.fixture
+def project() -> str:
+    # This test assumes billing is enabled on the project
```

Comment:
Please add a comment to clarify the reason for not enabling billing at the start of the test

Reply:
Fair point, if I can figure out if a service account can actually modify the billing account, I'll add the billing account as another secret and always enable bliling at the start.